core-bundle/public/backend.be7cab7b.js.LICENSE.txt

```diff
@@ -0,0 +1 @@
+/*! regenerator-runtime -- Copyright (c) 2014-present, Facebook, Inc. -- license (MIT): https://github.com/facebook/regenerator/blob/main/LICENSE */
```

Comment:
This seems to be unrelated to this pull request.

Reply:
Woopsy, no idea how it did get there… I'll remove it.